src/DonorDashboards/App.php

```diff
@@ -175,6 +176,8 @@ public function loadAssets()
             [],
             null
         );
+
+        do_action('give_donor_dashboard_enqueued_assets');
```

Comment:
Can we make this _enqueue_ instead of _enqueued_?

```suggestion
        do_action('give_donor_dashboard_enqueue_assets');
```